library/common/config/config.cc

```diff
@@ -46,6 +46,7 @@ const std::string config_header = R"(
 - &statsd_port 8125
 - &stream_idle_timeout 15s
 - &virtual_clusters []
+- &admin_interface {}
```

Comment:
I'm not certain this is safe. In fact, the interpretation of this when mapping YAML to Proto may result in an admin interface initialized to all (proto) default values, which could lead to crashes, security issues or unpredictable behavior.

Reply:
good point, removed